ng5/src/app/models/userService.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from './user'; 
+import { HttpClient } from '@angular/common/http';
+
+export class UserService {
+  readonly URL_SERVICE = 'https://wbooks-api-stage.herokuapp.com';
+
+  createUser(user: User, http:HttpClient) {
+    http.post<any>(this.URL_SERVICE + '/api/v1/users', {user}).toPromise()
```

Comment:
if this is the user service the default url will be 'this.URL_SERVICE + '/api/v1/users'. Add this as a variable to reuse it